code/modules/client/client_procs.dm

```diff
@@ -121,8 +121,17 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 		if(QDELETED(real_src))
 			return
 
+	//fun fact: Topic() acts like a verb and is executed at the end of the tick like other verbs. So we have to queue it if the server is
+	//overloaded
+	if(hsrc && hsrc != holder && TRY_QUEUE_VERB(src, .proc/Topic_wrapper, VERB_DEFAULT_QUEUE_THRESHOLD, hsrc, href, href_list))
+		return
 	..() //redirect to hsrc.Topic()
 
+///dumb workaround because byond doesnt seem to recognize the .proc/Topic() typepath for /datum/proc/Topic() from the client Topic,
+///so we cant queue it without this
```

Comment:
see click question, .proc will break here even if it wasn't internal

Reply:
see original review answer